### PR TITLE
New job adds for UXUI

### PR DESCRIPTION
**Pull request by jodiedoubleday**
### Description
- We have new roles available within the UXUI team, these are the two job adverts we are putting out.
- When adding these i noticed the most recent ads appear lower in the directory so i have reversed the loop.
- also lists had weird margins so I've moved them in a touch.

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by jackdcrawford** [comment]
Editor :raised_hands:

**Comment by jackdcrawford** [comment]
These look good :+1: 

I think that some logos for the tech used could be a nice addition?

File: _jobs/ux-designer.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: UX Designer
+tagline: Package - Up to &pound;40,000 per annum d.o.e + profit share + benefits
+date: 2015-10-14 12:00:00
+---
+We’re looking for UX designers to join us on the Holiday Extras campus and to become part of our expert team. The workload is fast-paced, but the atmosphere is fun.
+
+We know that UXers come in many different flavours - the person we are looking for will have the right mix of design thinking and practical skills, and is comfortable picking the right UX tools and activities for projects.  You will need to be pragmatic, proactive and understand the challenges and the opportunities that a large e-commerce business has to offer.  Your mission will be improving what we put in front of our customers, and you will have a passion for solving their problems.
+
+You’ll have a strong portfolio including mobile design and you’ll confidently work collaboratively to discover, generate and communicate new ideas.   You’ll be working alongside developers, copywriters and product owners to create awesome user experiences online; you should thrive on feedback, critique and iteration.
```
Comment (lukefrake) [inline _jobs/ux-designer.md]: tripple space after the .

**Comment by lukefrake** [comment]
1 little comment but other than that :+1:

**Comment by jodiedoubleday** [comment]
Im gonna get these out and add images after.

**Pull request merged**